corsheaders/models.py

```diff
@@ -3,3 +3,6 @@
 
 class CorsModel(models.Model):
     cors = models.CharField(max_length=255)
+    
+    def __unicode__(self)
```

Comment:
@adamchainz I need guidance on how to make the test. What and how should I test?